src/v2/Apps/Consign/Components/SoldRecently.tsx

```diff
@@ -1,11 +1,123 @@
 import React from "react"
-import { Box, Text } from "@artsy/palette"
+import { Flex, Spacer, Text } from "@artsy/palette"
+import { QueryRenderer, createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/Artsy"
+import { SoldRecentlyQuery } from "v2/__generated__/SoldRecentlyQuery.graphql"
+import { SoldRecently_targetSupply } from "v2/__generated__/SoldRecently_targetSupply.graphql"
+import { extractNodes } from "v2/Utils/ExtractNodes"
+import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
+import { Carousel } from "v2/Components/Carousel"
+import { SectionContainer } from "./SectionContainer"
+import { Media } from "v2/Utils/Responsive"
 
-export const SoldRecently: React.FC = () => {
+const HEIGHT = 300
+
+interface SoldRecentlyProps {
+  targetSupply: SoldRecently_targetSupply
+}
+
+export const SoldRecently: React.FC<SoldRecentlyProps> = ({ targetSupply }) => {
+  const recentlySoldArtworks = targetSupply.microfunnel.map(microfunnel => {
+    return extractNodes(microfunnel.artworksConnection)
+  })
+
+  return (
+    <SectionContainer>
+      <Text mb={[3, 2]} variant="largeTitle">
+        Sold recently on Artsy
+      </Text>
+      <Flex flexDirection="column">
+        <Carousel arrowHeight={HEIGHT}>
+          {recentlySoldArtworks.map(([artwork, _], key) => {
+            return (
+              <Flex
+                key={key}
+                flexDirection="column"
+                style={{ textAlign: "left" }}
+              >
+                <FillwidthItem
+                  artwork={artwork}
+                  hidePartnerName
+                  imageHeight={HEIGHT}
+                  showExtended={false}
+                />
+                {artwork.realizedPrice && (
+                  <>
+                    <Media greaterThanOrEqual="sm">
+                      <Flex flexDirection="row" alignItems="baseline">
+                        <Text variant="largeTitle">
```

Comment:
Can we give responsive props a try here and see how it looks?

```
variant={["caption", "caption", "largeTitle"]}
```

Reply:
What's the reason to use responsive props here? The layouts for the realized price line are different and will need a media query to render the right one.

Reply:
Ah, I didn't look close enough at the actual layout changes; thought it was just a size change 👍